build.gradle

```diff
@@ -182,3 +167,13 @@ wrapper {
 
 // Generate OSGI bundle metadata
 apply plugin: 'biz.aQute.bnd.builder'
+
+// gradle -PSFCI
+if (hasProperty('SFCI')) {
```

Comment:
Maybe we should do:

```
if (hasProperty('SFCI')) {
    apply from: 'sfci.gradle'
} else {
    apply from: 'evg.gradle'
}
```

so deploy continues to work for classic Evergage without having to know to pass a system property.

Reply:
Good call